src/rosdiscover/launch.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+__all__ = ('Launch',)
+
+from types import MappingProxyType
+from typing import Any, Mapping, Sequence
+
+from loguru import logger
+import attr
+import yaml
+
+@attr.s(frozen=True, slots=True, auto_attribs=True)
+class Launch:
+    """
```

Comment:
One-line description of what this data structure does/describes would be helpful 👍🏻

Reply:
done